app/views/users/show.html.erb

```diff
@@ -1,2 +1,90 @@
+        <div class="empty-div">
+        </div>
+      </div>
+      <div class="user-name">
+        <h1>
+          <%= @user.name %>
+        </h1>
+        <p>
+          Number of Posts: <%= @user.posts_counter%>
+        </p>
+      </div>
+  </div>
+  <div class="bio">
+    <div class="bio-title">
+      <h5>
+        Bio
+      </h5>
+    </div>
+    <div class="bio-text">
+      <p>
+        <%= @user.bio %>
+      </p>
+    </div>
+  </div>
+  <div class="post">
+    <div class="post-1-title post-title">
+      <h5>
+        Post #1
+      </h5>
+    </div>
+    <div class="post-1-text post-text">
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+      <div class="comments-likes">
+        <p class="comments-count">
+          Comments: X
+        </p>
+        <p class="likes-count">
+          Likes: Y
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="post">
+    <div class="post-2-title post-title">
+      <h5>
+        Post #2
+      </h5>
+    </div>
+    <div class="post-2-text post-text">
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+      <div class="comments-likes">
+        <p class="comments-count">
+          Comments: X
+        </p>
+        <p class="likes-count">
+          Likes: Y
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="post">
+    <div class="post-3-title post-title">
+      <h5>
+        Post #3
+      </h5>
+    </div>
+    <div class="post-3-text post-text">
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+      <div class="comments-likes">
+        <p class="comments-count">
+          Comments: X
+        </p>
+        <p class="likes-count">
+          Likes: Y
+        </p>
+      </div>
+    </div>
```

Comment:
You are hard coding the recent post from the users, but you need to use your method `recent_posts_from_user`. I read your text on the `Additional Information`. In order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.